agents/liquidation_agent.py

```diff
@@ -66,8 +74,94 @@ def __init__(
         self.borrow_address = borrow_address
         self.step = 0
 
+        # Uniswap variables
+        self.uniswap_pool_abi = uniswap_pool_abi
+        self.uniswap_pool_address = uniswap_pool_address
+        self.quoter_abi = quoter_abi
+        self.quoter_address = quoter_address
+        self.swap_router_abi = swap_router_abi
+        self.swap_router_address = swap_router_address
+        self.uniswap_fee = uniswap_fee
+
+        # balance of token a and token b
+        self.balance_collateral_asset = []
+        self.balance_debt_asset = []
+
+    def accountability(
+        self,
+        env,
+        liquidation_address: bytes,
+    ) -> bool:
+        """
+        Makes the accountability of a liquidation and returns a bool indicating
+        whether a liquidation is profitable or not
+        """
+        _debt_shares, collateral_assets = self.morpho_blue_abi.position.call(
+            env,
+            self.address,
+            self.morpho_blue_address,
+            [self.id_market, liquidation_address],
+        )[0][1:]
+        seized_assets = collateral_assets // 2
+        liquidation_call_event = self.morpho_blue_abi.liquidate.call(
+            env,
+            self.address,
+            self.morpho_blue_address,
+            [self.market_params, liquidation_address, seized_assets, 0, b""],
+        )[1]
+        if liquidation_call_event:
```

Comment:
Is this a boolean value? Or are you checking it's not None?

Reply:
Just removed this if in a new commit.